config/secrets.yml

```diff
@@ -12,11 +12,18 @@
 
 development:
   secret_key_base: 5091e17fd4c0a3ddacdc134a3926e29c464cdaf0526684912fcd61f89c30680ec5be714f3f66b2c6c7ed6c5384b60eaf31584e10b9cff8e7bdaf9e2d8d6d11ab
-
+  ezid_default_shoulder: doi:10.5072/FK2
+  ezid_user: apitest
+  ezid_password: apitest
 test:
   secret_key_base: 450fc6b88e8a27fe9ca23621599762b9faab14421121131d98f6f4de737cd8a74266fd7e2c0f7316a2388066722ea01e354bb9930f4ea223bd7bd02cf11e65c3
-
+  ezid_default_shoulder: doi:10.5072/FK2
+  ezid_user: apitest
+  ezid_password: apitest
 # Do not keep production secrets in the repository,
 # instead read values from the environment.
 production:
-  secret_key_base: <%= ENV["SECRET_KEY_BASE"] %>
+  secret_key_base: <%= ENV['SECRET_KEY_BASE'] %>
+  ezid_default_shoulder: <%= ENV['EZID_DEFAULT_SHOULDER'] %>
+  ezid_user: <%= ENV['EZID_USER'] %>
+  ezid_password: <%= ENV['EZID_PASSWORD'] %>
```

Comment:
Rails 4.1 introduced a very easy way to manage environmental variables in your app instead of using an external gem to do this. Simply define your secret keys in your environments, then simple call 

```
Rails.application.secrets.<key>
```

In your code.